packages/core/src/components/button/_common.scss

```diff
@@ -345,6 +352,13 @@ $button-intents: (
     background: none;
     cursor: not-allowed;
     color: $pt-dark-text-color-disabled;
+
+    &:active,
+    &.pt-active {
```

Comment:
there's an identical selector on line 332 in this same block. can we merge them? why wasn't that done in the first place?

Reply:
Is it fine now?